workspaces/cogito-ethereum/source/CogitoEthereum.js

```diff
@@ -24,11 +24,16 @@ class CogitoEthereum {
     return new Web3(cogitoProvider)
   }
 
-  getTelepathChannel = ({ channelId: id, channelKey: key, appName }) => {
-    return this.telepath.createChannel({ id, key, appName })
+  getTelepathChannel = async ({ channelId: id, channelKey: key, appName }) => {
+    const channel = await this.telepath.createChannel({ id, key, appName })
+    if (process.env.NODE_ENV !== 'test') {
```

Comment:
It would be nice if the same behaviour could be accomplished by using the jest __mocks__ construction.